devito/compiler.py

```diff
@@ -257,7 +271,8 @@ def make(loc, args):
                 log.write(" ".join(command))
                 log.write("\n\n")
                 try:
-                    subprocess.check_call(command, stderr=err, stdout=log)
+                    #subprocess.check_call(command, stderr=err, stdout=log)
```

Comment:
Debug leftover?

Reply:
I'm experiencing a weird, YASK-related (ie, not involving devito at all) failure on Travis only. Those are just there because I had to hand an output msg to Chuck to investigate. Will drop them all and perform a final clean up pass before merging